docs/rules/valid-typeof.md

```diff
@@ -1,6 +1,6 @@
 # enforce comparing `typeof` expressions against valid strings (valid-typeof)
 
-For a vast majority of use cases, the result of the `typeof` operator is one of the following string literals: `"undefined"`, `"object"`, `"boolean"`, `"number"`, `"string"`, `"function"` and `"symbol"`. It is usually a typing mistake to compare the result of a `typeof` operator to other string literals.
+For a vast majority of use cases, the result of the `typeof` operator is one of the following string literals: `"undefined"`, `"object"`, `"boolean"`, `"bigint"`, `"number"`, `"string"`, `"function"` and `"symbol"`. It is usually a typing mistake to compare the result of a `typeof` operator to other string literals.
```

Comment:
does it make sense to move `bigint` to after `symbol`, as it was newly added?

and adding something like [further reading](https://github.com/eslint/eslint/blob/master/docs/rules/prefer-arrow-callback.md#further-reading) wouldn't hurt. :)

Reply:
OK, I moved `bigint` to the end of the applicable lists. I also added a further reading section, which links to the MDN `typeof` documentation.